app/helpers/requests/application_helper.rb

```diff
@@ -120,14 +98,13 @@ def recap_print_only_input(requestable)
       end
     end
 
-    # rubocop:disable Style/NumericPredicate
     def enum_copy_display(item)
       display = ""
       display += item[:enum_display] unless item[:enum_display].nil?
       display += " " if !item[:enum_display].nil? && !item[:copy_number].nil?
       # For scsb materials
       display += item[:enumeration] if item[:enumeration]
-      display += "Copy #{item[:copy_number]}" unless item[:copy_number].nil? || (item[:copy_number]) == 0 || item[:copy_number] == 1 || item[:copy_number] == '1'
+      display += "Copy #{item[:copy_number]}" unless item[:copy_number].nil? || (item[:copy_number]).zero? || item[:copy_number] == 1 || item[:copy_number] == '1'
```

Comment:
we should keep (item[:copy_number]) == 0. see https://github.com/pulibrary/requests/pull/514/files

Reply:
@christinach updated the commit.  Not sure why  that happened.  maybe the -a removed it or maybe the issue was behind the keyboard.

Reply:
most likely the -a